test/integration/user/api/interactivity.test.js

```diff
@@ -225,15 +225,15 @@ describe('Interactivity', () => {
                 });
 
                 it('should not fire a featureEnter event when the mouse is moved inside the same feature', done => {
```

Comment:
Does this description need to be updated?

Reply:
No, the code changes just to start listening to the events before the user actions.